scripts/test/nightly/runTests.sh

```diff
@@ -22,29 +22,31 @@ MAVEN_DIR=$(cd "$SCRIPTS_DIR" && cd ../java && pwd)
 
 pushd "$SCRIPTS_DIR/test"
 
-if [ "$#" -lt 3 ] || [ "$#" -gt 4 ]; then
-  echo "Usage: $0 <vpc> <subnet> <results bucket> <optional-test-type>"
-  echo "Default test type is functional, valid test types are: performance, functional"
+if [ "$#" -ne 4 ]; then
+  echo "Usage: $0 <vpc> <subnet> <results bucket> <test-type>"
+  echo "Valid test types are: performance, functional"
```

Comment:
This change will break the crontab, as it doesn't specify the test type for functional tests.